### PR TITLE
Add search input option for text input component

### PR DESCRIPTION
**Pull request by burrata**
#### Related issues
https://meetup.atlassian.net/browse/MUP-10166

screen shot 2017-04-13 at 6 08 29 pm

### CODE REVIEW

File: src/forms/TextInput.jsx [hunk 1]
```diff
@@ -47,17 +53,18 @@ class TextInput extends React.Component {
 
 		return (
 			<div>
-				<label className={labelClassNames} htmlFor={other.id}>
+				<label className={labelClassNames} htmlFor={id}>
```
Comment (eilinora) [inline src/forms/TextInput.jsx]: 🎉

File: src/forms/TextInput.jsx [hunk 2]
```diff
@@ -47,17 +53,18 @@ class TextInput extends React.Component {
...
 					{label}
 				</label>
 
-				<input type='text'
+				<input type={this.getInputType()}
```
Comment (eilinora) [inline src/forms/TextInput.jsx]: i think you can write this as a one liner vs a secondary function :)

**Comment by eilinora** [comment]
Overall looks good, small change request :)

**Comment by burrata** [comment]
@eilinora if the pr changes look good to you, lemme know or feel free to merge!

**Pull request merged**